### ISSUE TITLE
Unusable course meta

### ISSUE THREAD

**Issue by bynicolas** [LearnPress/LearnPress#160]
You have a few metas saved with a course or lesson.

Namely, I refer to `_lp_duration`.  My issue here is that you save to the database the time **and**  unit in the **same** meta.

This becomes quickly unusable. 

1. I cannot translate the display of the unit once it's saved to the database.
2. The units are saved in singular form everytime.
3. I cannot compute cumulative time for each lessons since I have mixed type in the database.

So I propose to save the values of `_lp_duration` in one field and the type of unit in another field.

And so we are not burdened with the translation there. I propose we go a bit further and we save all values in seconds. This way, we can display the correct unit at runtime in the language we need to display it into.
Plus, going this route, we save a meta field as we don't need to save the units to the database.

The unit field can still be used in the admin area as a way to convert `2 hour` into `7200` seconds just before saving to the DB

**Comment by bynicolas** [comment]
Digging through the code, I saw that you used the `_lp_duration` meta and just type converted to `absint()`.

I guess that works but it gets rather messy!

I need to check the value to know what unit I'm working with before knowing if it's `30 minutes`, `30 hours` or `30 days` that I'm adding together.

I managed to get this going, but I still vote for changing the way this data is stored in the DB so it's always in `seconds`. This way it would be less manipulation of the data to know what we are working with!
